frappe_email_send_override/hooks.py

```diff
@@ -1,28 +1,26 @@
-import frappe
-
-app_name = "email_send_override"
-app_title = "Email Send Override"
+app_name = "frappe_frappe_email_send_override"
```

Comment:
@zeel-codder This should be `frappe_email_send_override` right?

Reply:
yes